spec/cassettes/get_user_info_by_api_token_with_invalid_bio.yml

```diff
@@ -0,0 +1,141 @@
+---
+http_interactions:
+- request:
+    method: get
+    uri: https://api.instagram.com/v1/users/45364550?client_id=<API_TOKEN>
+    body:
+      encoding: US-ASCII
+      string: ''
+    headers:
+      User-Agent:
+      - Faraday v0.9.1
+  response:
+    status:
+      code: 200
+      message: OK
+    headers:
+      Server:
+      - nginx
+      Date:
+      - Thu, 09 Apr 2015 11:54:22 GMT
+      Content-Type:
+      - application/json; charset=utf-8
+      X-Ratelimit-Remaining:
+      - '4383'
+      Content-Language:
+      - en
+      Expires:
+      - Sat, 01 Jan 2000 00:00:00 GMT
+      Vary:
+      - Cookie, Accept-Language, Accept-Encoding
+      X-Ratelimit-Limit:
+      - '5000'
+      Pragma:
+      - no-cache
+      Cache-Control:
+      - private, no-cache, no-store, must-revalidate
+      Set-Cookie:
+      - csrftoken=e90f95afede17a1bda9a4ccde0379d24; expires=Thu, 07-Apr-2016 11:54:22
+        GMT; Max-Age=31449600; Path=/
+      Connection:
+      - keep-alive
+      Content-Length:
+      - '376'
+    body:
+      encoding: UTF-8
+      string: '{"meta":{"code":200},"data":{"username":"yoh_miyhu","bio":"be myself \ud83d\udc8e\ud83d\ud83d\ude0f","website":"","profile_picture":"https://scontent.cdninstagram.com/t51.2885-19/11906329_960233084022564_1448528159_a.jpg","full_name":"\u6c6a\u795dSallyanne","counts":{"media":0,"followed_by":0,"follows":208},"id":"45364550"}}'
```

Comment:
here you can see two(`\\ud83d`) in `bio`'s value **be myself \ud83d\udc8e`\ud83d\ud83d`\ude0f** which was causing error while parsing using `Oj.load`